CIP-????/README.md

```diff
@@ -0,0 +1,205 @@
+implementation detail (unlike the format of blocks).
+
+### The ledger-script interface
+
+The ledger and Plutus scripts have a common interface, described in [CIP-35].
+CIPs relating to this inteface are relevant to both the ledger and to the Plutus CIP categories.
+
+## Specification
+
+### What merits a ledger CIP?
+
+The criterion for deciding if a change to the ledger merits a CIP is as follows:
+changes to the ledger require going through the CIP process whenever
+every implementation of the Cardano ledger needs to be standardized on the details.
+
+Bug fixes are an exception to this criterion, they do not merit a CIP except in the case
+that the fix is substantially complicated.
+A "bug fix" is a change to behavior where:
+- The implemented behavior does not match the specification; or
+- The specified behavior is clearly wrong (in the judgment of relevant experts)
+
+Serialization changes are another possible exception to the criterion.
+Many serialization changes can be handled as a part of the normal development process
+without the need for a CIP.
+Dramatic changes to the serialization, however, may benefit from the CIP process.
+
+The ledger rules MUST be standardized in order for consensus to be maintained,
+but things like the ledger events are more open to debate.
+
+Changes to the protocol parameter values do not require a CIP since they are
+a governance issue (see [CIP-1694]).
+
+### Ledger Core CIP registry
+
+Any CIP which proposes a change to the ledger MUST be added to this table:
+
+| #    | Title                                       | Status   |
+|---   | ---                                         | ---      |
+|   31 | Reference inputs                            | Active   |
+|   32 | Inline datums                               | Active   |
+|   33 | Reference scripts                           | Active   |
+|   59 | Terminology Surrounding Core Features       | Active   |
+|   80 | Transaction Serialization Deprecation Cycle | Proposed |
+| 1694 | A proposal for entering the Voltaire phase  | Proposed |
+
+### Expectations for ledger CIPs
+
+* Familiarity with the existing ledger specifications is required to propose changes to the ledger.
+* The CIP specifications for ledger CIPs must be sufficiently detailed for inclusion in
+  a formal ledger specification.
+* Some changes may require getting an expert opinion.
+  For example, changes to the incentives model (such as changing the reward calculation)
+  should include a proper game theoretic analysis.
+
+## Rationale
+
+### There is only one implementation, why limit the scope of ledger CIPs in this way?
+
+Even though there is currently only one implementation, this provides us with a clear
+definition of what is essential to the ledger.
+It also provides a clear path for future implementations.
+
+### Why is the specification vague about the role of ledger events in the CIP process?
+
+This decision should be left to the community as more use cases emerge.
```

Comment:
I still think ledger events should pretty clearly be "out". They don't affect the operation of the Cardano network. A competing implementation could omit them and be perfectly correct.